pkg/task/meta.go

```diff
@@ -314,14 +319,14 @@ func (m *metaTaskExecutor) asRollbackInstance(objectName string) (*metaTaskExecu
 		return nil, true, err
 	}
 
-	var repeater repeatWithResourceExecutor
-	if m.repeater.isRepeat() {
-		repeater = newRepeatWithResourceExecByObjectNames(objectName)
-	}
+	//var repeater repeatWithResourceExecutor
+	//if m.repeater.isRepeat() {
+	//	repeater = newRepeatWithResourceExecByObjectNames(objectName)
+	//}
 
 	return &metaTaskExecutor{
 		metaTask:   rbMT,
 		identifier: i,
-		repeater:   repeater,
+		//repeater:   repeater,
```

Comment:
Repeater is required. However, repeater capability for rollback has been removed as part of rollback fix. So a repeater will be used with MetaTask for execution, and if they are put based execution, a rollback if that ever happens will be taken care of via other means rather than via repeater.

The other means, is to make rollback logic understand the concept of multiple resources that got created via repeater. This is made possible by adding the object name to objectName property upon every resource creation via repeater.

This may not or does not sound to be the fool proof approach. However, we shall tackle this re-design at later/future iterations.